mods/default/mapgen.lua

```diff
@@ -1527,10 +1475,8 @@ local mg_name = minetest.get_mapgen_setting("mg_name")
 if mg_name == "v6" then
 	default.register_ores()
 	default.register_mgv6_decorations()
-	minetest.register_on_generated(default.generate_nyancats)
 elseif mg_name ~= "singlenode" then
 	default.register_biomes()
 	default.register_ores()
 	default.register_decorations()
-	minetest.register_on_generated(default.generate_nyancats)
```

Comment:
See https://github.com/minetest/minetest_game/pull/1190#issuecomment-231827129
Because singlenode cannot be selected in world creation ```if mg_name ~= "singlenode"``` has no effect so nyancats (and other lines here) were never disabled in singlenode.
This might be a good reason to bring singlenode back to world creation?